src/components/header/index.css

```diff
@@ -38,6 +38,10 @@
   background: rgba(0, 0, 0, 0.08);
 }
 
+.dark-background {
```

Comment:
Can we use the right css pseudo class instead?

Reply:
can you tell me which pseudo class should I include?
Should I hover and click and unhover ?

Reply:
Maybe that the best solution. We could use some css to make the change stay for a few milliseconds.

Reply:
I think we cannot apply hover effect programatically (i tried) , so instead of making an extra class (.dark-background), should i apply styles using js?

Reply:
Nah, an extra class is okay. How about we share the definition for a pseudo class click as well as the fake click?